src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -2729,6 +2729,23 @@ public GHContent getReadme() throws IOException {
         return requester.withUrlPath(getApiTailUrl("readme")).fetch(GHContent.class).wrap(this);
     }
 
+    /**
+     * Create repo variable
+     *
+     * @param name
+     *            the name
+     * @throws IOException
+     *             the io exception
+     */
+    public void createRepoVariable(final GHRepositoryVariable name) throws IOException {
+        root().createRequest()
+                .withUrlPath(getApiTailUrl("actions/variables"))
+                .method("POST")
+                .with("name", name.getName())
+                .with("value", name.getValue())
+                .send();
+    }
+
```

Comment:
Sigh, and while you're at it please deprecate the `getRepoVariable()` method?  It should either be named `getRepositoryVariable` or `getVariable`. :sob:

Reply:
I think that getVariable is better option